packages/simm-validation/test/in-range.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { inRange } from "../../simm-helpers/src/in-range";
+import { expect, describe, test } from "vitest";
+
+describe("inRange", () => {
+  test("returns true for values within the range", () => {
```

Comment:
thêm mấy case check kiểu Number, số âm, số thập phân
255; // two-hundred and fifty-five
255.0; // same number
255 === 255.0; // true
255 === 0xff; // true (hexadecimal notation)
255 === 0b11111111; // true (binary notation)
255 === 0.255e3; // true (decimal exponential notation)